psi4/driver/procrouting/proc.py

```diff
@@ -3464,21 +3465,30 @@ def format_vector(label, data):
         lines += [""]
         core.print_out("\n".join(lines) + "\n")
 
+    gauge = core.get_option("ADC", "GAUGE").lower()
     computed = {}
     if any(prop in properties for prop in ("TRANSITION_DIPOLE", "OSCILLATOR_STRENGTH")):
         data = state.transition_dipole_moments
         computed["Transition dipole moment (in a.u.)"] = data
-        adc_wfn.set_variable(name + " transition dipoles", core.Matrix.from_array(data))
+        adc_wfn.set_variable(f"{name} transition dipoles", core.Matrix.from_array(data))
 
     if "OSCILLATOR_STRENGTH" in properties:
-        data = state.oscillator_strengths.reshape(-1, 1)
-        computed["Oscillator strength (length gauge)"] = data
-        adc_wfn.set_variable(name + " oscillator strengths", core.Matrix.from_array(data))
+        if gauge == "velocity":
+            data = state.oscillator_strengths_velocity.reshape(-1, 1)
+        else:
+            data = state.oscillator_strengths.reshape(-1, 1)
+        computed[f"Oscillator strength ({gauge} gauge)"] = data
+        adc_wfn.set_variable(f"{name} oscillator strengths", core.Matrix.from_array(data))
```

Comment:
can the variable include the gauge like https://github.com/psi4/psi4/blob/master/psi4/src/psi4/cc/ccresponse/optrot.cc#L442-L514